index.html

```diff
@@ -64,6 +64,201 @@ <h1>I’m Hussen. Glad to see you!</h1>
+            </div>
+            <div class="blocks">
+              <h2 class="project-title">Multi-Post Stories</h2>
+              <div class="tframes">
+                <h3><b>CANOPY</b></h3>
+                <ol class="frames-list">
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ol>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tag-list">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="blocks-button">See Project</button>
+            </div>
+          </div>
+        </div>
+        <!-- card works  3-->
+        <div class="work-container">
+          <div class="cards">
+            <div class="snapshot-portofolio">
+              <img src="img/work-img/Snapshoot Portfolio 3.png" alt="SP" />
+            </div>
+            <div class="blocks">
+              <h2 class="project-title">Tonic</h2>
+              <div class="tframes">
+                <h3><b>CANOPY</b></h3>
+                <ol class="frames-list">
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ol>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tag-list">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="blocks-button">See Project</button>
+            </div>
+          </div>
+        </div>
+        <!-- card works  4-->
+        <div class="work-container">
+          <div class="cards">
+            <div class="snapshot-portofolio">
+              <img src="img/work-img/Snapshoot Portfolio 4.png" alt="SP" />
+            </div>
+            <div class="blocks">
+              <h2 class="project-title">Multi-Post Stories</h2>
+              <div class="tframes">
+                <h3><b>CANOPY</b></h3>
+                <ol class="frames-list">
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ol>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tag-list">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="blocks-button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-section">
+        <div class="about-Container">
+          <div class="text-1">
+            <h1>About Myself</h1>
+          </div>
```

Comment:
- [ ] For the about section, note the following changes in reference to the figma template:
- Kindly adjust the font of the `About Myself` to match what's given 👍 
- The `LET'S CONNECT` subheader color needs to be changed to `#7f8cff` as given on the template 
- Nice work with the button, but you need to adjust the width and height to match the design given 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/227728376-08d9a49a-8a68-48e8-b480-71d95148a70c.png)


## Figma:
![image](https://user-images.githubusercontent.com/50990925/227728334-7774af1b-3127-4415-81fc-bc6f166c6582.png)